app/assets/javascripts/event-creation-app.js

```diff
@@ -37,7 +37,15 @@ function buildEventCreationApp(elementId, diagram) {
 
         edgesToRemove.forEach(edge => {
           diagram.edges.remove(edge.id);
+
+          const nextStepIndex = fromNode.step.next_steps.findIndex(thisStep => {
+            return thisStep.id === edge.to;
+          });
+
+          fromNode.step.next_steps.splice(nextStepIndex, 1);
```

Comment:
This fixes a bug on the editor, when un-linking steps from each other. Previously, the step remained linked when you 'fired' it.